Changes

```diff
@@ -185,6 +185,9 @@ Working version
 - MPR#7695, GPR#1541: Fatal error: exception Ctype.Unify(_) with field override
   (Jacques Garrigue, report by Nicolas Ojeda Bar)
 
+- MPR#7702, GPR#1553: refresh raise counts when inlining a function
+  (Vincent Laviron, Xavier Clerc, report by Cheng Sun, review by Gabriel Scherer)
```

Comment:
please remove my name, I haven't reviewed this PR -- I merely passed @lthls' approval.

Reply:
OK; I thought the reviewer was the one adding the green badge to a PR.

Reply:
The green badge should only happen after serious review work has been performed, and in general the two people coincide (reviewing and badging), but not in this particular case. In general "review by" is useful to credit everyone who contributed useful comments during the review period.

Reply:
Understood; could you squash/merge?